server/buildbuddy_server/buildbuddy_server.go

```diff
@@ -990,6 +992,10 @@ type bsLookup struct {
 }
 
 func getBestFilename(filename, blobname string) string {
+	if strings.HasPrefix(filename, "//") && !strings.HasPrefix(filename, "///") {
+		// This is not a normal file path, let the implementation handle it
+		return filename
+	}
```

Comment:
This is somewhat redundant w/ my earlier comment re. RPC service, but I think we shouldn't introduce "special" bytestream URIs which actually map to blobstore resources, since blobstore is not accessible via the bytestream API. A separate `blobtype=buildlog` URL param would be cleaner, IMO; that way we are keeping bytestream and blobstore separate.

Reply:
works for me.

Reply:
```suggestion
```